parquet-column/src/main/java/org/apache/parquet/column/statistics/BinaryStatistics.java

```diff
@@ -46,8 +46,8 @@ public void mergeStatisticsMinMax(Statistics stats) {
 
   @Override
   public void setMinMaxFromBytes(byte[] minBytes, byte[] maxBytes) {
-    max = Binary.fromByteArray(maxBytes);
-    min = Binary.fromByteArray(minBytes);
+    max = Binary.fromUnmodifiedByteArray(maxBytes);
+    min = Binary.fromUnmodifiedByteArray(minBytes);
```

Comment:
Do we know that this is always called with constant byte buffers and not reused byte buffers?